apps/sms/services/js/moz_mobile_message/messaging_service.js

```diff
@@ -0,0 +1,193 @@
+/*global BridgeServiceMixin, Settings */
```

Comment:
I'm not sure either. The reason why I put them into single folder is because I want centralize the messaging API related services into single place instead of disperse to separated folders, but I'm fine with other possible solution once we have agreement. @julienw wdyt?